sequencer/src/l1_client.rs

```diff
@@ -204,4 +231,46 @@ mod test {
 
         Ok(())
     }
+
+    #[async_std::test]
+    async fn test_get_pending_deposits() -> anyhow::Result<()> {
+        // how many deposits will we make
+        let deposits = 5;
+
+        let anvil = Anvil::new().spawn();
+        let address = anvil.addresses().first().cloned().unwrap();
+        dbg!(&address);
+
+        let l1_client = L1Client::new(anvil.endpoint().parse().unwrap());
+
+        let wallet: LocalWallet = anvil.keys()[0].clone().into();
+
+        // In order to deposit we need a provider that can sign.
+        let provider =
+            Provider::<Http>::try_from(anvil.endpoint())?.interval(Duration::from_millis(10u64));
+        let client = SignerMiddleware::new(provider, wallet.with_chain_id(anvil.chain_id()));
+        let client = Arc::new(client);
+
+        // Initialize a contract with some deposits
+        let contract = contract_bindings::fee_contract::FeeContract::new(address, Arc::new(client));
```

Comment:
Currently it doesn't look like you are deploying the contract.

Reply:
The contract bindings have a `deploy` method. The `new` method is to attach to an already deployed contract.

Reply:
:facepalm:

Reply:
fixed in https://github.com/EspressoSystems/espresso-sequencer/pull/1188/commits/340b63e0151cc1435fc387ccd6acd7ab0a7b56ed